assembl/static2/js/app/app.jsx

```diff
@@ -23,28 +28,44 @@ class App extends React.Component {
   }
 
   componentDidUpdate() {
-    const { debate, location, params } = this.props;
+    const { debate, location, params, timeline } = this.props;
     if (!params.phase && !debate.debateLoading && location.pathname.split('/').indexOf('debate') > -1) {
-      const currentPhaseIdentifier = getCurrentPhaseIdentifier(debate.debateData.timeline);
+      const currentPhaseIdentifier = getCurrentPhaseIdentifier(timeline);
       browserHistory.push(get('debate', { slug: params.slug, phase: currentPhaseIdentifier }));
     }
   }
 
   render() {
+    const { timelineLoading, timeline, putTimelineInStore } = this.props;
     const { debateData, debateLoading, debateError } = this.props.debate;
     const divClassNames = classNames('app', { 'harvesting-mode-on': this.props.isHarvesting });
+    if (!timelineLoading) {
```

Comment:
as you use `withLoadingIndicator` you probably never get here with `timelineLoading === true`. If it happens, there is a problem

Reply:
yes, there is a problem... I don't know why. I'll look at it again